### ISSUE TITLE
WebP Gimp 2.9 Patch

### ISSUE THREAD

**Issue by draekko** [nathan-osman/gimp-webp#8]
I've prepared a patch to submit to the folks at Gimp to have webp be supported for 2.9+  natively. Just giving you a heads up. Attaching the patch with this comment. Doing some more last minute tests to make sure i have it all right before doing so. Will probably submit after supper today.

[gimp-webp-patch.zip](https://github.com/nathan-osman/gimp-webp/files/407344/gimp-webp-patch.zip)

**Comment by nathan-osman** [comment]
That's awesome! If they are willing to accept the plugin in Gimp, there will no longer be a real need to maintain the plugin here.

**Comment by draekko** [comment]
Once i do i'll cc the link to you so you can have some input over there as well. Hopefully it gets accepted without too much fuss, they're rather picky on formatting and stuff like that.

**Comment by draekko** [comment]
Done, see this link for what i wrote https://bugzilla.gnome.org/show_bug.cgi?id=769651 _crossing fingers_

**Comment by nathan-osman** [comment]
Excellent. I've subscribed myself to the bug.

**Comment by draekko** [comment]
hopefully it'll be a minimum of hassles to get it in there.

**Comment by draekko** [comment]
It's in the GIMP git repository Micheal Natterer pushed it this afternoon, w00t! Now i need to just finish updating my patch to fix exiv2 to work with this :)

**Comment by nathan-osman** [comment]
@draekko great job! This is exciting.

**Comment by draekko** [comment]
Indeed. So now it will be available natively in 2.9.x, just need to worry about updates for 2.8 :)

**Comment by draekko** [comment]
Here's the commit entry https://git.gnome.org/browse/gimp/commit/?id=c1021a3e3a8d4574505c8008ea619018a8fb1b69

**Comment by draekko** [comment]
Ha check this out https://plus.google.com/+gimp/posts/R2kA4STMQNa

**Comment by draekko** [comment]
and this https://www.facebook.com/gimpofficial/photos/a.1704355829778238.1073741828.1700570406823447/1800507783496375/?type=3

**Comment by nathan-osman** [comment]
Thanks for sharing!
